addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -133,14 +133,39 @@
                             <small t-field="event.organizer_id" t-options="{'widget': 'contact', 'fields': ['phone', 'mobile', 'email'], 'with_microdata': True,}"/>
                         </div>
                         <!-- Social -->
-                        <div class="o_wevent_sidebar_block">
-                            <h6 class="o_wevent_sidebar_title">Share</h6>
-                            <p>Find out what people see and say about this event, and join the conversation.</p>
-                            <t t-snippet-call="website.s_share">
-                                <t t-set="_no_title" t-value="True"/>
-                                <t t-set="_classes" t-valuef="o_wevent_sidebar_social mx-n1"/>
-                                <t t-set="_link_classes" t-valuef="o_wevent_social_link"/>
-                            </t>
+                        <div class="oe_structure">
+                            <section>
+                                <div class="o_wevent_sidebar_block">
+                                    <h6 class="o_wevent_sidebar_title">Share</h6>
+                                    <p>Find out what people see and say about this event, and join the conversation.</p>
+                                </div>
+                                <div class="s_share text-start o_editable" data-snippet="s_share">
+                                    <a href="https://www.facebook.com/sharer/sharer.php?u={url}"
```

Comment:
I saw that we have a text-decoration on hover on each link. Could perhaps be a good idea to add a css rule for the `s_share_*` classes to remove it. Could also be great to have a hover background change like we had before.